### DISCUSSION TITLE
Is WebDriver BiDi Supported in Serenity/JS

### DISCUSSION THREAD

**Discussion by ahmed-al-sadoon** [serenity-js/serenity-js#2533] [How to... ?] [answered]
Greetings,

I have hird about WebDriver BiDi and wanted to ask if its supported in Serenity/JS

**Comment by jan-molak** [comment, marked as answer]
Serenity/JS supports [WebDriver BiDi](https://webdriver.io/blog/2024/08/15/webdriverio-v9-release/) via its integration with [WebdriverIO 9](https://serenity-js.org/handbook/test-runners/webdriverio/).
